### PR TITLE
Hospital data

### PR DESCRIPTION
**Pull request by aakritsubedi**
- Display hospital data from CSV(Google Drive)
- ToDo: initially hospitalData is empty, search hospital name in the search field and the list appears **fix**

### CODE REVIEW

File: src/components/Dashboard/HospitalList.vue [hunk 1]
```diff
@@ -20,133 +22,60 @@
...
+        }
+      }
+
+      return objArray;
+    }
+  },
+  async created() {
+    const hospitals = await api.fetchHospital();
+    this.hospitals = await this.convertToJSON(hospitals);
+  },
```
Comment (samirsilwal) [inline src/components/Dashboard/HospitalList.vue]: use store instead

**Review by samirsilwal — CHANGES_REQUESTED** [review]

File: src/api/index.js [hunk 2]
```diff
@@ -67,28 +70,38 @@ export default {
...
+      isp: localStorage.getItem("isp"),
     };
   },
   countryData: async () => {
     const url = "https://api.covid19api.com/summary";
     try {
-      let countries = await axios.get(url).then(data => data );
+      let countries = await axios.get(url).then((data) => data);
 
       return { countries };
```
Comment (samirsilwal) [inline src/api/index.js:92]: use throw

File: src/api/index.js [hunk 3]
```diff
@@ -67,28 +70,38 @@ export default {
...
-  }
+  },
+  fetchHospital: async () => {
+    const url = "http://localhost:9091/data/HospitalData";
+    try {
+      let hospitals = await axios.get(url);
+
+      return hospitals.data;
+    } catch (error) {
+      return error;
```
Comment (samirsilwal) [inline src/api/index.js]: .

**Review by samirsilwal — CHANGES_REQUESTED** [review]

File: src/components/Dashboard/HospitalList.vue (hunk 1 above)
Comment (aakritsubedi) [reply src/components/Dashboard/HospitalList.vue]: :ok:

**Pull request merged**